options.cpp

```diff
@@ -60,7 +60,7 @@ namespace
         {"number-processes", 1, 0, 205},
         {"drop", 0, 0, 206},
         {"unlogged", 0, 0, 207},
-        {"flat-nodes",1,0,209},
+        {"flat-nodes",1,0, 'F'},
```

Comment:
Is this the formatting clang-format wants?

Reply:
With regard to the space in front of the option: yes. However, if I run clang-format over that particular change it will reformat the entire array (the indention is wrong) and I'd rather leave that as is.